.circleci/config.yml

```diff
@@ -32,28 +32,28 @@ checkout: &checkout
   run:
     name: "Checkout code"
     command: |
+      function retry_10() {
+        # Retries up to 10 times with 10 second intervals
+        for i in $(seq 1 10); do
+          "$@" && return || sleep 10
+        done
+        echo "$@ failed after $retries attempts"
+        exit 1
+      }
       cd $HOME
-      mkdir -p .ssh
-      chmod 0700 .ssh
-      ssh-keyscan -t rsa github.com >> .ssh/known_hosts
```

Comment:
How come we no longer need this?

Reply:
We don't do anything private. As long as we use HTTPS and not ssh or git, this seems to just work.

Reply:
No other use of SSH in our stack I think